src/connection.cpp

```diff
@@ -221,6 +221,14 @@ namespace oxen::quic
         return "{:02x}"_format(fmt::join(std::begin(data), std::begin(data) + datalen, ""));
     }
 
+    void Connection::halt_events()
+    {
+        log::trace(log_cat, "{} called", __PRETTY_FUNCTION__);
+        event_del(packet_io_trigger.get());
+        event_del(packet_retransmit_timer.get());
```

Comment:
These are `event_ptr` objects which which calls event_del on destruction, so should just be `.reset()` instead of having `event_del` called directly.